pkgs/applications/misc/gnuradio-ais/default.nix

```diff
@@ -26,11 +26,15 @@ stdenv.mkDerivation rec {
     done
   '';
 
+  patches = [ ./fix-volk.patch ];
```

Comment:
https://github.com/bistromath/gr-ais/pull/18/commits/8502d0252a2a1a9b8d1a71795eaeb5d820684054

Should I update the file to point to my PR commit?